src/inverseRobot.js

```diff
@@ -25,6 +25,16 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const res = {};
+
+  for (const i in robot) {
```

Comment:
Use more meaningful names, "i" not a good name, something like "key" would be fine